src/domain/payments/index.types.d.ts

```diff
@@ -1,6 +1,6 @@
 type PriceRatio = {
-  convertFromUsd(convert: UsdPaymentAmount): BtcPaymentAmount
-  convertFromBtc(convert: BtcPaymentAmount): UsdPaymentAmount
+  convertFromUsd(convert: UsdPaymentAmount): BtcPaymentAmount | ValidationError
```

Comment:
How can a validation error occur here? This seems wrong.

Reply:
It's because there was math and that `number` type inside that method. Maybe we do need to do the math -> PriceRatio change in this PR as well to keep the types clean.

Reply:
yes I think the math fix has to be in here as well.